source/common/http/utility.h

```diff
@@ -621,6 +622,9 @@ AuthorityAttributes parseAuthority(absl::string_view host);
 envoy::config::route::v3::RetryPolicy
 convertCoreToRouteRetryPolicy(const envoy::config::core::v3::RetryPolicy& retry_policy,
                               const std::string& retry_on);
+
+bool isZeroRttSafeRequest(Http::RequestHeaderMap& request_headers);
```

Comment:
nit: I think there is no need for "ZeroRtt" in the name of this method since the definition of "safe" in this context is from rfc7231 which is oblivious to zero RTT / early data stuff.